acceptance-tests/src/test/resources/hudson.tasks.Maven.MavenInstaller

```diff
@@ -0,0 +1,29 @@
+{
+    "list": [
+        {
+            "id": "3.8.6",
+            "name": "3.8.6",
+            "url": "https://repo.maven.apache.org/maven2/org/apache/maven/apache-maven/3.8.6/apache-maven-3.8.6-bin.zip"
+        },
+        {
+            "id": "3.6.3",
+            "name": "3.6.3",
+            "url": "https://repo.maven.apache.org/maven2/org/apache/maven/apache-maven/3.6.3/apache-maven-3.6.3-bin.zip"
+        },
+        {
+            "id": "3.5.4",
+            "name": "3.5.4",
+            "url": "https://repo.maven.apache.org/maven2/org/apache/maven/apache-maven/3.5.4/apache-maven-3.5.4-bin.zip"
+        },
+        {
+            "id": "3.3.9",
+            "name": "3.3.9",
+            "url": "https://repo.maven.apache.org/maven2/org/apache/maven/apache-maven/3.3.9/apache-maven-3.3.9-bin.zip"
+        },
+        {
+            "id": "3.2.5",
```

Comment:
Can we also add Maven `3.0.5` and `3.1.1`?

Reply:
Done. Just to clarify a little bit. It is not the complete list of maven versions that we can use. It's the initial, cached list, that is available immediately after the startup. After the list is fetched from the update center all versions will be available.

Reply:
Ok thanks! So we don't actually use all these versions in tests right?

Reply:
Only the latest one `3.8.6` at the moment

Reply:
Yes, we can parametrize tests and run with multiple versions of maven. The only thing is that these tests are really slow. If we will be implementing our CI and run tests nightly, for example, then we can test all versions.